tests/unit/orchestrators/test_cache_utils.py

```diff
@@ -242,8 +242,8 @@ def test_fetching_cached_step_run_queries_cache_candidates(
     mock_list_run_steps = mocker.patch(
         "zenml.client.Client.list_run_steps",
         return_value=Page(
-            page=1,
-            size=1,
+            index=1,
+            max_size=1,
```

Comment:
Don't we need both? How many items are on this page, and whats the maximum size allowed per page?